docs/content/en/docs/pipeline-stages/healthcheck.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Deploy Healthchecks"
+linkTitle: "Deploy Healthchecks"
+weight: 50
+featureId: deploy.status_check
+aliases: [/docs/how-tos/healthchecks]
+---
+
+This page describes how skaffold runs healthchecks for deployed resources - waits for them to stabilize, and reports errors on failure.
+
+### Overview
+
+Commands that trigger a deployment like `skaffold dev`, `skaffold deploy`, `skaffold run`, and `skaffold apply` perform a `healthcheck` for select Kubernetes resources and wait for them to be stable.
```

Comment:
Don't use verbatim mode for "healthcheck" as it risks confusion with Docker's image `HEALTHCHECK` extension.
```suggestion
Commands that trigger a deployment, like `skaffold dev`, `skaffold deploy`, `skaffold run`, and `skaffold apply`, monitor select Kubernetes resources and wait for them to become ready.
```